public/stylesheets/analyzer.css

```
@@ -0,0 +1,947 @@
+  --analyzer-primary: #6366f1;
+  --analyzer-primary-dark: #4f46e5;
+  --analyzer-primary-light: #eef2ff;
+  --analyzer-success: #10b981;
+  --analyzer-warning: #f59e0b;
+  --analyzer-danger: #ef4444;
+  --analyzer-surface: #ffffff;
+  --analyzer-muted: #64748b;
+  --analyzer-border: #e2e8f0;
+  --analyzer-shadow: 0 4px 6px -1px rgba(0, 0, 0, 0.08), 0 2px 4px -2px rgba(0, 0, 0, 0.06);
+  --analyzer-radius: 12px;
+  max-width: 1100px;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)